openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/common/PaginationAndFilterTest.java

```diff
@@ -82,6 +82,45 @@ public void checkFlikerInFilter() throws Exception {
     }
   }
 
+  @RepeatedIfExceptionsTest(repeats = 2)
+  @Order(2)
+  public void noDataPresentWithFilter() throws Exception {
+    Thread.sleep(waitTime);
+    Events.click(webDriver, By.cssSelector("[data-testid='closeWhatsNew']")); // Close What's new
+    Thread.sleep(waitTime);
+    Events.click(webDriver, By.cssSelector("[data-testid='tables']")); // Tables
+    Events.click(webDriver, By.cssSelector("[data-testid='checkbox'][id='BigQuery']")); // Select Filter
+    try {
+      WebElement noDataFound = wait.until(ExpectedConditions.presenceOfElementLocated(By.xpath(
+          "//*[contains(text(), 'No matching data assets found')]")));
+      if (noDataFound.isDisplayed()) {
+        throw new Exception("Data not found with filter count more than 0");
+      }
+    } catch (TimeoutException exception) {
+      LOG.info("Success");
+    }
+  }
+
+  @RepeatedIfExceptionsTest(repeats = 2)
+  @Order(3)
+  public void dataPresentWithFilter() throws Exception {
+    Thread.sleep(waitTime);
```

Comment:
do we really need `waitTime` here?

Reply:
Removed. Addressed in the latest commit.